charts/cofide-agent/templates/agent-service.yaml

```diff
@@ -14,7 +13,7 @@ metadata:
     {{- toYaml . | nindent 4 }}
   {{- end }}
 spec:
-  type: {{ .Values.xdsServer.service.type | default "ClusterIP" }}
+  type: {{ .Values.xdsServer.service.type |default .Values.xdsServer.service.type }}
```

Comment:
Default is the same?

Reply:
It's the same but I wanted to ensure it was sourced from the values file - previously it was hardcoded in this template.

Reply:
So can we just remove the default?